### PR TITLE
added indicator fields

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/31072

## Description
added indicator fields for main indicators use

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@ytauman Hi have you went over this?

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Behavior.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+   "type": "shortText",
+   "width": 150
+  }
+ ],
+ "content": true,
+ "defaultRows": [
+  {
+
+  }
+ ],
```
Comment (ShahafBenYakir) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-Behavior.json:37]: What is this?

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Community_Notes.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  {
+
+  },
+  {
+
+  },
+  {
+
+  }
+ ],
```
Comment (ShahafBenYakir) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-Community_Notes.json:43]: What is this?

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Publications.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+ "caseInsensitive": true,
+ "cliName": "publications",
+ "closeForm": false,
+ "columns": [
+  {
+   "displayName": "Title",
+   "fieldCalcScript": "",
+   "isDefault": true,
+   "isReadOnly": false,
+   "key": "Title",
```
Comment (ytauman) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-Publications.json:12]: should be lower-cased

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Publications.json [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+   "selectValues": null,
+   "type": "shortText",
+   "width": 304
+  },
+  {
+   "displayName": "Source",
+   "fieldCalcScript": "",
+   "isDefault": true,
+   "isReadOnly": false,
+   "key": "Source",
```
Comment (ytauman) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-Publications.json:24]: should be lower-cased

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Publications.json [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+   "selectValues": null,
+   "type": "shortText",
+   "width": 198
+  },
+  {
+   "displayName": "Link",
+   "fieldCalcScript": "",
+   "isDefault": true,
+   "isReadOnly": false,
+   "key": "Link",
```
Comment (ytauman) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-Publications.json:36]: should be lower-cased

File: Packs/CommonTypes/IndicatorFields/indicatorfield-Publications.json [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+   "selectValues": null,
+   "type": "shortText",
+   "width": 208
+  },
+  {
+   "displayName": "Timestamp",
+   "fieldCalcScript": "",
+   "isDefault": true,
+   "isReadOnly": false,
+   "key": "Timestamp",
```
Comment (ytauman) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-Publications.json:48]: should be lower-cased

**Review by ytauman — COMMENTED** [review]
general comment - noticed that all the new fields that were added have a name of `indicatorfield-<field_name>.json` while the rest have a name of `incidentfield-<field_name>.json`.
i'm guessing this is a legacy thing, just making sure it's ok..

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**